docs/community-helpers.md

```diff
@@ -5,7 +5,7 @@ editLink: false
 ---
 
 # Community Helpers
-> Share your helpers at our [Wiki Page](https://github.com/codeceptjs/CodeceptJS/wiki/Community-Helpers)
+> Share your helpers at our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Community-Helpers)
```

Comment:
need to pull up a new path for documentation codeceptjs
and in other places too

Reply:
Thanks, updated!